contents/docs/feature-flags/evaluation-environments.mdx

```diff
@@ -3,16 +3,16 @@ title: Evaluation environments
 sidebar: Docs
 showTitle: true
 availability:
-  free: partial
-  selfServe: full
+  free: false
+  selfServe: boost
   enterprise: full
```

Comment:
I don't think this is used anymore and setting `selfServe: boost` is breaking the build, so maybe just delete?